packages/storybook/cxl-ui/cxl-playbook-accordion/theme=cxl-playbook.stories.js

```diff
@@ -0,0 +1,50 @@
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-app-layout.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import faqData from '../cxl-vaadin-accordion/theme=cxl-faq.data.json';
+import '@vaadin/vaadin-ordered-layout';
+
+export default {
+  title: 'CXL UI/cxl-playbook-accordion',
+};
+
+export const CXLPlaybookAccordionTheme = () => html`
+  <style>
+    .plural .entry-title {
+      margin: 0;
+    }
+  </style>
+
+    <cxl-playbook-accordion
+      id="cxl-vaadin-accordion-26107"
+      theme="reverse"
+      class="cxl-playbook-accordion-class"
+    >
+      ${faqData.map(
+        (el) => html`
+          <vaadin-accordion-panel
+            id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+            class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+            theme="reverse"
+          >
+            <vaadin-horizontal-layout theme="padding" slot="summary">
+              <div class="left">
+                <vaadin-checkbox value="Option" theme="custom"></vaadin-checkbox>
```

Comment:
`theme` attribute names aren't random.

All our custom theme names are prefixed `cxl`, and descriptive of what they're about.

In this case, `theme="cxl-playbook-accordion"` would be appropriate.

Reply:
Since we don't use checkbox label here at all, this theme and matching style got dropped.